package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hmcts/cmc-common-frontend",
-  "version": "1.40.0",
+  "version": "1.41.0",
```

Comment:
Why you made that change? Is that needed for that PR? If so update yarn.lock.

Reply:
<img width="1256" alt="screen shot 2018-03-16 at 08 20 49" src="https://user-images.githubusercontent.com/8955824/37510459-0606ed32-28f3-11e8-804a-99a5071eb45d.png">

it is to match the prototype and it came up in conversation with marc and tarik 
https://hmcts-cmc-prototype.herokuapp.com/claimant/full/c-4/task-list/check-and-send?check=2